core/src/test/java/org/elasticsearch/common/breaker/MemoryCircuitBreakerTests.java

```diff
@@ -235,6 +235,9 @@ public void testConstantFactor() throws Exception {
             fail("should never reach this");
         } catch (CircuitBreakingException cbe) {
             assertThat("breaker was tripped exactly twice", breaker.getTrippedCount(), equalTo(2L));
+
+            long newUsed = (long)(breaker.getUsed() * breaker.getOverhead());
+            assertThat(cbe.getMessage().contains("would be [" + newUsed "/"), equalTo(true));
```

Comment:
This is missing a `+` between `newUsed` and `"/"`, so it's not compiling currently